### ISSUE TITLE
Missing Morph in Amos 6:14 and Job 30:24

### ISSUE THREAD

**Issue by HeyITGuyFixIt** [openscriptures/morphhb#79]
I was using a script to loop through each word in the your XML files and it had an issue with the following two words:

Job.30.24 : <w lemma="l/2004" morph="HC" id="18rfH">לָהֶ֥ן</w>

Amos.6.14 : <w lemma="m/l/935" morph="HR/Np" id="30HAz">מִ/לְּב֥וֹא</w>

It looks like the lamed in each word should be divided from the word, as is reflected in the lemma. The Hebrew text content and the morph do not reflect this.

If I am correct about this, these two words should be corrected to:

Job.30.24 : <w lemma="l/2004" morph="HR/HC" id="18rfH">לָ/הֶ֥ן</w>

Amos.6.14 : <w lemma="m/l/935" morph="HR/HR/Np" id="30HAz">מִ/לְּ/ב֥וֹא</w>

**Comment by jdejoode** [comment]
I've looked at Job.30.24 and indeed the parsing is wrong there. Good catch. It is a preposition (lamed) with a suffix, according to the ETCBC database: https://bibleol.3bmoodle.dk/text/show_text/ETCBC4/Iob/30/24 I think the latter is correct. The parsing wouldn't be the one your proposed (as its a suffix and not a conjunction), but you are right that it needs a change:

What about the following? (Note that this changes both lemma and morph)

Job.30.24 : `<w lemma="l" morph="HR/Sp3fp" id="18rfH">לָ/הֶ֥ן</w>`

**Comment by jdejoode** [comment]
I've looked at Amos.6.14 and once again, you're right that the parsing is wrong, but the parsing solution is a bit more complex here than in the example from Job. [ETCBC](https://bibleol.3bmoodle.dk/text/show_text/ETCBC4/Amos/6/14) reads two prepositions and the infinitive construct here. That makes more sense. It would therefore be:

Amos.6.14 : <w lemma="m/l/935" morph="HR/R/Vqc" id="30HAz">מִ/לְּ/ב֥וֹא</w>

BUT a) this also affects other cases, b) this might be part of a composed place name.

A. This also affects other occurrences, I only cite two

    <verse osisID="2Chr.7.8">
    <w lemma="m/935" morph="HR/Np" id="1459X">מִ/לְּב֥וֹא</w>
    </verse>

and 

    <verse osisID="2Kgs.14.25">
    <w lemma="m/935" morph="HR/Ncmsc" id="12vQR">מִ/לְּב֥וֹא</w>
    </verse>

Other occurrences include: 1Kgs.8.65, Ezek.47.20, Josh.13.5, Judg.3.3, Num.13.21, Num.34.8. Often here לְּב֥וֹא is parsed a proper noun (as you proposed as well).

B. The Dictionary of Classical Hebrew has a lemma for Lebo-hamath as a placename. HALOT does something similar (vol 1, p. 515), but mentions a counter-example in 1Chr 5:9 (עַד־לְבֹ֣וא מִדְבָּ֔רָה) where לְבֹ֣וא could also means something like 'entrance/border'. 

It would probably be good to be consistent and either think of Lebo-hamath as a place name, or to think of it as an infinitive construct + place name. It is an interesting construction. Often the lebo (itself already containing a preposition) is accompanied by another preposition indicating directionality.

**Comment by joeldruark** [comment]
I agree with Johan here that the lemma should be changed in Job 30:24, either to "l" (with a pronominal suffix instead of the independent personal pronoun) or to "3860".  Johan prefers the former, I would prefer the latter on the basis of HALOT.  That is, it looks to me like HALOT does not see this word as a preposition with a suffix but rather as its own word (with a note that the text here is corrupted and should read differently altogether).  It's a judgment call, I suppose.  It sure looks like ETCBC reads it one way, and HALOT reads it the other.  But in either case, something needs to be fixed in Job 30:24.

**Comment by DavidTroidl** [comment]
Job.30.24 has been updated to 3860.

**Comment by jdejoode** [comment]
Also see https://github.com/openscriptures/morphhb/pull/102
